app/services/message_templates/hook_execution_service.rb

```diff
@@ -15,13 +15,16 @@ def perform
 
   def trigger_templates
     # TODO: let's see whether this is needed and remove this and related logic if not
```

Comment:
```suggestion
   
```

let's remove the comment as it don't make sense anymore

Reply:
Removed